tensorflow-core/pom.xml

```diff
@@ -74,6 +72,9 @@
       <properties>
         <javacpp.platform>${os.name}-${os.arch}</javacpp.platform>
       </properties>
```

Comment:
This should be removed as well, as it will be set in the top properties on this file